lib/recipes-data/src/lib/extract_recipedata_from_element.test.ts

```diff
@@ -0,0 +1,375 @@
+import {AssetType} from "@guardian/content-api-models/v1/assetType";
+import type {Block} from "@guardian/content-api-models/v1/block";
+import {ElementType} from "@guardian/content-api-models/v1/elementType";
+import {response} from "./../res/recipejson-example.json";
+import {extractRecipeData} from "./extract_recipedata_from_element";
+
+describe("extractRecipeData", () => {
+
+  it("should work when block containing single recipe elements ", () => {
+    const webUrl = response.results[0].webUrl
```

Comment:
this should be the `id` field not the `webUrl` field.... you might as well just put it in as a static string though and drop the `recipejson-example.json`